themes/root/templates/RecordDriver/EDS/export-ris.phtml

```diff
@@ -0,0 +1,108 @@
+<?
+$items = $this->driver->getItems();
```

Comment:
I'm happy to help brainstorm this if you'd like me to take a closer look. We can live with it as-is if we have to, but if we can at least reuse some of the base code, that would probably be beneficial.